Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -0,0 +1,1519 @@
+                    'varying vec4 tile_featureColor; \n' +
+                    'void main() \n' +
+                    '{ \n' +
+                    '    tile_main(); \n' +
+                    '    vec2 st = computeSt(' + batchIdAttributeName + '); \n' +
+                    '    vec4 featureProperties = texture2D(tile_batchTexture, st); \n' +
+                    '    float show = ceil(featureProperties.a); \n' +      // 0 - false, non-zeo - true
+                    '    gl_Position *= show; \n';                          // Per-feature show/hide
+                if (handleTranslucent) {
+                    newMain +=
+                        '    bool isStyleTranslucent = (featureProperties.a != 1.0); \n' +
+                        '    if (czm_pass == czm_passTranslucent) \n' +
+                        '    { \n' +
+                        '        if (!isStyleTranslucent && !tile_translucentCommand) \n' + // Do not render opaque features in the translucent pass
+                        '        { \n' +
+                        '            gl_Position *= 0.0; \n' +
+                        '        } \n' +
+                        '    } \n' +
+                        '    else \n' +
+                        '    { \n' +
+                        '        if (isStyleTranslucent) \n' + // Do not render translucent features in the opaque pass
+                        '        { \n' +
+                        '            gl_Position *= 0.0; \n' +
+                        '        } \n' +
+                        '    } \n';
+                }
+                newMain +=
+                    '    tile_featureColor = featureProperties; \n' +
+                    '}';
+            } else {
+                newMain =
+                    'varying vec2 tile_featureSt; \n' +
+                    'void main() \n' +
+                    '{ \n' +
+                    '    tile_main(); \n' +
+                    '    tile_featureSt = computeSt(' + batchIdAttributeName + '); \n' +
+                    '}';
+            }
+
+            return renamedSource + '\n' + getGlslComputeSt(that) + newMain;
+        };
+    };
+
+    function getHighlightOnlyShader(source) {
+        source = ShaderSource.replaceMain(source, 'tile_main');
+        return source +
+               'void tile_color(vec4 tile_featureColor) \n' +
+               '{ \n' +
+               '    tile_main(); \n' +
+               '    gl_FragColor *= tile_featureColor; \n' +
+               '} \n';
+    }
+
+    function modifyDiffuse(source, diffuseUniformName) {
+        // If the glTF does not specify the _3DTILESDIFFUSE semantic, return a basic highlight shader.
+        // Otherwise if _3DTILESDIFFUSE is defined prefer the shader below that can switch the color mode at runtime.
+        if (!defined(diffuseUniformName)) {
+            return getHighlightOnlyShader(source);
+        }
+
+        // Find the diffuse uniform
```

Comment:
Given regex can be confusing, a more descriptive example of what we're matching, or some hardcoded representations would go a long way here, or if you're feeling fancy, you can just link to https://regexper.com/#%2Funiform%5Cs%2B(vec%5B34%5D%7Csampler2D)%5Cs%2BdiffuseUniformName%3B%2F